snippets/worker-latency-profile-intro.adoc

```diff
@@ -25,6 +25,6 @@ This behavior can cause problems if your network is prone to latency issues, esp
 
 To avoid this problem, you can use _worker latency profiles_ to adjust the frequency that the `Kubelet` and the Kubernetes Controller Manager wait for status updates before taking action. These adjustments help to ensure that your cluster runs properly if network latency between the control plane and the worker nodes is not optimal.
 
-These worker latency profiles contain three sets of parameters that are pre-defined with carefully tuned values to control the reaction of the cluster to increased latency. No need to experimentally find the best values manually.
+These worker latency profiles contain three sets of parameters that are predefined with carefully tuned values to control the reaction of the cluster to increased latency. No need to experimentally find the best values manually.
```

Comment:
If you're touching up the docs, I might suggest rewording the last sentence to sound a tad more formal:
```suggestion
These worker latency profiles contain three sets of parameters that are predefined with carefully tuned values to control the reaction of the cluster to increased latency. There is no need to experimentally find the best values manually.
```

Reply:
Great suggestion, @skopacz1 , glad to do that!